File: udemy/java/03_estrutura_condicional/exercicios/Exercicio2.java

import java.util.Scanner;

public class Exercicio2 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int numero;

        System.out.print("Digite um número: ");
        numero = sc.nextInt();

        if (numero % 2 == 0) {
            System.out.println("Numero: " + numero + " = Par");
        } else {
            System.out.println("Numero: " + numero + " = Impar");
        }

    }

}

File: udemy/php/09- funcoes/3- parametros_funcoes/exercicio2.php
<?php


function numeroParOuImpar($x) {
    if ($x % 2 == 0) {
        echo "O número $x é par <br>";
    } else {
        echo "O número $x é impar <br>";
    }
}


numeroParOuImpar(2);
numeroParOuImpar(3);